src/main/fc/fc_core.c

```diff
@@ -109,6 +109,8 @@ int16_t magHold;
 int16_t headFreeModeHold;
 
 static bool reverseMotors = false;
```

Comment:
It looks like `reverseMotors` is no longer used. If so, can you remove it and `isMotorsReversed` function.